docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +90,160 @@ services:
           editable: false
         EOF
         /run.sh
-    image: grafana/grafana:latest
+    image: grafana/grafana:11.0.0
     ports:
       - "3000:3000"
 ```
 
 After running `docker-compose up`, open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
 
-The tutorials are designed to be followed in order and generally build on each other. Each example explains what it does and how it works.
+## Configure {{< param "PRODUCT_NAME" >}}
+
+Alloy is configured through `config.alloy` file which contains a set of components. Components do basic things 
+like identifying which logs we want to scrape, how we want to process them, and where we will send them. 
+Our configuration will connect components into a workflow.
```

Comment:
I propose we replace the lines 102-104 with the following:

Once the local Grafana instance has been set up, the next step is to give detailed instructions to Alloy on which logs we want to scrape, how we want to process that data, and where we want the data sent. This is done through `config.alloy` file which contains components with instructions for Alloy to execute. Our configuration will connect components into a workflow. 

The examples run on a single host so that you can run them on your laptop or in a Virtual Machine. You are encouraged to try the examples using a `config.alloy` file and experiment with the examples yourself.